lib/runner_test.go

```diff
@@ -373,155 +378,265 @@ func TestShuffleOrder(t *testing.T) {
 func TestRunSingleTestSuccess(t *testing.T) {
 	t.Parallel()
 
-	r := setupDefaultRunner()
-
+	r, _, _ := setupDefaultRunner()
 	testFolder, _ := filepath.Abs("../testdata/success")
-	testResult := r.runSingleTest("hello_world_test.sh", testFolder)
-	expected := TestResult{
-		TestFile: "hello_world_test.sh",
-		Success:  true,
-		ExitCode: 0,
-		Output:   "Hello World!\n",
+	testFile := "hello_world_test.sh"
+	testResult := r.runSingleTest(testFile, testFolder)
+	if testResult.TestFile != testFile {
+		t.Fatalf("\nExpected TestFile: %v\nHave: %v\n", testFile, testResult.TestFile)
 	}
-	if !reflect.DeepEqual(testResult, expected) {
```

Comment:
Curious where did this go?

Reply:
Output is an `io.Reader` now and doesn't work well with `reflect.DeepEqual`. I'm performing the comparison of the fields individually.